typescript/infra/scripts/middleware/circle-relayer.ts

```diff
@@ -27,14 +32,18 @@ async function check() {
   );
   const addresses = readJSON(dir, 'addresses.json');
   const contracts = attachContractsMap(addresses, liquidityLayerFactories);
+
   const app = new LiquidityLayerApp(
     contracts,
     multiProvider,
-    bridgeAdapterConfigs,
+    config.liquidityLayerConfig,
   );
 
   while (true) {
-    for (const chain of [Chains.goerli, Chains.fuji]) {
+    for (const chain of Object.keys(
+      // @ts-ignore Can't figure how to make it a type predicate
+      objFilter(config.liquidityLayerConfig, (_, config) => !!config.circle),
```

Comment:
What's the error? You may be missing a `: config is LiquidityLayerConfig` type annotation on the filter lambda func

Reply:
That did the trick!